index.js

```diff
@@ -94,8 +94,20 @@ var initConfig = function(projectConfig) {
  * The singleton public agent. This is the public API of the module.
  */
 var publicAgent = {
+  // FOR_TESTING_ONLY
+  __DISABLE_START_CHECK__: false,
```

Comment:
I like your solution better that what I had.  I've changed the trace agent to align with the debug agent.